torch/__init__.py

```diff
@@ -1693,5 +1693,20 @@ def _sparse_coo_tensor_unsafe(*args, **kwargs):
 # Register MPS specific decomps
 torch.backends.mps._init()
 
+if not _running_with_deploy():
+    class _TritonLibrary(object):
```

Comment:
nit: we should move this to another file, maybe `torch.backends.triton`?

Reply:
@nikitaved - Can you address this one?